operator/src/main/resources/scripts/utils.sh

```diff
@@ -889,3 +889,81 @@ function adjustPath() {
     fi
   fi
 }
+
+#
+# checkCommonMount
+#   purpose: If the COMMON_MOUNT_PATH directory exists, it echoes the contents of output files
+#            in ${COMMON_MOUNT_PATH}/commonMountLogs dir. It returns 1 if a SEVERE message
+#            is found in any of the output files in ${COMMON_MOUNT_PATH}/commonMountLogs dirs.
+#            It also returns 1 if 'successfully' message is not found in the output files
+#            or if the COMMON_MOUNT_PATH directory is empty. Otherwise it returns 0 (success).
+#            See also 'commonMount.sh'.
+#
+function checkCommonMount() {
+  # check commonMount results (if any)
+  if [ ! -z "$COMMON_MOUNT_PATH" ]; then
+    trace FINE "Common Mount: COMMON_MOUNT_PATH is '$COMMON_MOUNT_PATH'."
+    traceDirs $COMMON_MOUNT_PATH
+    touch ${COMMON_MOUNT_PATH}/testaccess.tmp
+    if [ $? -ne 0 ]; then
+      trace SEVERE "Common Mount: Cannot write to the COMMON_MOUNT_PATH '${COMMON_MOUNT_PATH}'" && return 1
+    fi
+
+    out_files=$(set -o pipefail ; ls -1 $COMMON_MOUNT_PATH/commonMountLogs/*.out 2>1 | sort --version-sort) \
+      || (trace SEVERE "Common Mount: Assertion failure. No files found in '$COMMON_MOUNT_PATH/commonMountLogs/*.out" \
+      && return 1)
+    severe_found=false
+    for out_file in $out_files; do
+      if [ "$(grep -c SEVERE $out_file)" != "0" ]; then
+        trace SEVERE "Common Mount: Error found in file '${out_file}' while initializing commonMount."
+        severe_found=true
+      elif [ "$(grep -c successfully $out_file)" = "0" ]; then
+        trace SEVERE "Common Mount: Command execution was unsuccessful in file '${out_file}' while initializing commonMount."
+        severe_found=true
+      else
+        trace "Common Mount: Contents of '${out_file}':"
+        trace "Common Mount: End of '${out_file}' contents"
+      fi
+      cat $out_file
+    done
+    [ "${severe_found}" = "true" ] && return 1
+    rm -fr $COMMON_MOUNT_PATH/commonMountLogs
+    [ -z "$(ls -A $COMMON_MOUNT_PATH)" ] \
+      && trace SEVERE "Common Mount: No files found in '$COMMON_MOUNT_PATH'. Do your commonMount images have files in their '$COMMON_MOUNT_PATH' directories?" \
```

Comment:
1 - For more context, consider adding "This path is configurable using the domain resource 'serverPod.commonMount.mountPath' attribute."

2 - Long SEVERE lines can be broken up into multiple strings:
```
trace SEVERE  "foo " \
              "bar " \
              "baz"
```